PowerUp/app/src/androidTest/java/powerup/systers/com/powerup/espresso/AvatarRoomActivityTest.java

```diff
@@ -0,0 +1,226 @@
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
+        onView(withId(R.id.eyeRight)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+
+        onView(withId(R.id.hairLeft)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
+        onView(withId(R.id.hairLeft)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+
+        onView(withId(R.id.hairRight)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
+        onView(withId(R.id.hairRight)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+
+        onView(withId(R.id.faceLeft)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
+        onView(withId(R.id.faceLeft)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+
+        onView(withId(R.id.faceRight)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
+        onView(withId(R.id.faceRight)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+
+        onView(withId(R.id.clotheLeft)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
+        onView(withId(R.id.clotheLeft)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+
+        onView(withId(R.id.clotheRight)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
+        onView(withId(R.id.clotheRight)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+
+        onView(withId(R.id.continueButtonAvatar)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
+        onView(withId(R.id.continueButtonAvatar)).check(matches(isDisplayed()));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+
+    }
+
+    @Test
+    public void clickEvents() {
+
+
+        onView(withId(R.id.eyeLeft)).perform(click());
+        onView(withId(R.id.eyes)).check(matches(new DrawableMatcher(R.drawable.eye3)));
```

Comment:
It would be good to here check the entire cycle to left and right as the last step is a corner case